apps/daimo-mobile/src/logic/device.ts

```diff
@@ -0,0 +1,53 @@
+import { isDERPubKey, assert } from "@daimo/common";
+import { Hex, isHex, keccak256 } from "viem";
+
+const emojis = [
+  "🍏",
+  "🍎",
+  "🍐",
+  "🍊",
+  "🍋",
+  "🍌",
+  "🍉",
+  "🍇",
+  "🍓",
+  "🍈",
+  "🍒",
+  "🍑",
+  "🍍",
+  "🥭",
+  "🍅",
+  "🍆",
+  "🥑",
+  "🥝",
+  "🥥",
+  "🥭",
+  "🍠",
+  "🥔",
+  "🥕",
+  "🌽",
+  "🌶️",
+  "🍄",
+  "🥒",
+  "🥦",
+  "🍞",
+  "🥖",
+  "🥐",
+];
```

Comment:
hmm given that there's a canonical list of keys

we could report the position in the list, maybe Device A / B / C

Reply:
If devices are removed and added the positions shift which might create confusion.

Reply:
replaced by slots/A/B/C now